spec/dom-to-image-more.spec.js

```diff
@@ -587,17 +598,17 @@
             function assertTextRendered(lines) {
                 return function () {
                     return new Promise(function (resolve, reject) {
-                        Tesseract.recognize(canvas()).then(function (result) {
+                        Tesseract.recognize(canvas(), 'eng').then((data) => {
```

Comment:
Tesseract 4.x has much better language support, and all our tests are in English, so might as well be explicit